client/app/css/less/ovh-ui-kit/icon.less

```diff
@@ -4,4 +4,18 @@
     &.oui-icon_small {
         font-size: 1.4rem;
     }
+
+    &.oui-icon_progress-info-foreground-color {
```

Comment:
Why do you put this code here? It should be managed by ui-kit no?

Reply:
We have to discuss to have css colors helpers (background-color / color) in ui-kit.